J Boss.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+<title>J Boss Community</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" type="text/css" href="bootstrap/css/bootstrap.css">
+<!-- Optional Bootstrap theme -->
+<link rel="stylesheet" href="bootstrap/css/bootstrap-theme.css">
+
+<style type="text/css">
+  .bs-example { 
+   margin: 20px; 
+  } 
+</style>
+
+
+</head>
+<body>
+	<style>
+.how
+{
+	height: 100vh;
+	background-size: cover;
+	background-position: center;
+}
+.hell{
+float:center; 
```

Comment:
You should never add css in html files. In angular we have <component-name>.component.css and hence you should make your css changes in the same file. For all frontend projects we are using angular and hence follow the guidelines 
for the same.